src/extension.ts

```diff
@@ -9,66 +9,93 @@
  */
 import * as vscode from "vscode";
 import * as devCommands from "./liberty/devCommands";
+import * as path from "path";
 
 import { LibertyProject, ProjectProvider } from "./liberty/libertyProject";
+import { LanguageClient, LanguageClientOptions, Executable, ServerOptions } from "vscode-languageclient";
+import { workspace, commands, ExtensionContext, extensions, window, StatusBarAlignment, TextEditor, languages } from "vscode";
 
+const LANGUAGE_CLIENT_ID = "LANGUAGE_ID_LIBERTY";
+
+let languageClient: LanguageClient;
+const SUPPORTED_LANGUAGE_IDS = ["properties", "plaintext"];
 export async function activate(context: vscode.ExtensionContext): Promise<void> {
-	
-	let projectProvider = ProjectProvider.getInstance();
-	if ( !projectProvider ) {
-		projectProvider = new ProjectProvider(context);
-		ProjectProvider.setInstance(projectProvider);
-	}
-
```

Comment:
this should there

Reply:
This code block was moved to the `registerCommands` function below. Can you create a code suggestion with the extra lines that are needed?

Reply:
Updated in https://github.com/OpenLiberty/liberty-tools-vscode/pull/148